src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtCasesToNomisIntTest.kt

```diff
@@ -3218,227 +3218,259 @@ class CourtCasesToNomisIntTest : SqsIntegrationTestBase() {
+                .withRequestBody(matchingJsonPath("sentences[1].sentenceCategory", equalTo("2013")))
+                .withRequestBody(matchingJsonPath("sentences[1].sentenceCalcType", equalTo("FTR_14")))
+                .withRequestBody(matchingJsonPath("sentences[1].active", equalTo("true")))
+                .withRequestBody(matchingJsonPath("returnToCustody.returnToCustodyDate", equalTo("2025-04-23")))
+                .withRequestBody(matchingJsonPath("recallRevocationDate", equalTo("2025-04-01")))
+                .withRequestBody(matchingJsonPath("returnToCustody.recallLength", equalTo("14")))
+                .withRequestBody(matchingJsonPath("returnToCustody.enteredByStaffUsername", equalTo("T.SMITH"))),
+            )
           }
-          val rawMessage = fromNomisCourtSentencingQueue.readRawMessages().first()
-          val sqsMessage: SQSMessage = rawMessage.fromJson()
 
-          assertThat(sqsMessage.Type).isEqualTo("courtsentencing.resync.sentence-adjustments")
-          val request: SyncSentenceAdjustment = sqsMessage.Message.fromJson()
-          assertThat(request.offenderNo).isEqualTo(OFFENDER_NO)
-          assertThat(request.sentences).hasSize(1)
-          assertThat(request.sentences[0].sentenceId.offenderBookingId).isEqualTo(BOOKING_ID)
-          assertThat(request.sentences[0].sentenceId.sentenceSequence).isEqualTo(2)
-          assertThat(request.sentences[0].adjustmentIds).contains(1L, 2L)
-        }
+          @Test
+          fun `will create mapping for each breach appearance created for recall`() {
+            courtSentencingMappingApi.verify(
+              postRequestedFor(urlEqualTo("/mapping/court-sentencing/court-appearances/recall"))
+                .withRequestBody(matchingJsonPath("dpsRecallId", equalTo("dc71f3c5-70d4-4faf-a4a5-ff9662d5f714")))
+                .withRequestBody(matchingJsonPath("nomisCourtAppearanceIds[0]", equalTo("101")))
+                .withRequestBody(matchingJsonPath("nomisCourtAppearanceIds[1]", equalTo("102"))),
+            )
+          }
 
-        @Test
-        fun `will create success telemetry`() {
-          verify(telemetryClient).trackEvent(
-            eq("recall-inserted-success"),
-            check {
-              assertThat(it["recallType"]).isEqualTo("FTR_14")
-              assertThat(it["dpsRecallId"]).isEqualTo("dc71f3c5-70d4-4faf-a4a5-ff9662d5f714")
-              assertThat(it["offenderNo"]).isEqualTo(OFFENDER_NO)
-              assertThat(it["dpsSentenceIds"]).isEqualTo("9ee21616-bbe4-4adc-b05e-c6e2a6a67cfc, 7ed5c261-9644-4516-9ab5-1b2cd48e6ca1")
-              assertThat(it["nomisSentenceSeq"]).isEqualTo("1, 2")
-              assertThat(it["nomisBookingId"]).isEqualTo("$BOOKING_ID")
-            },
-            isNull(),
-          )
+          @Test
+          fun `will send message to nomis migration for each sync sentence adjustments that have been activated`() {
```

Comment:
only functional change is shown here;

previously we had two adjustments on same sentence generate two sync messages with both in the body of the message
now we send two messages with each adjustment in each message. The end result will be the same.